docs/extract.html

```
@@ -173,6 +175,15 @@ <h2>The CLI shape</h2>
                     dictionary) as preferred subject/object spellings;
                     non-restrictive, off by default; toggling re-extracts
                     (ADR 0014)
+--redact [GROUP]    mask secrets and pattern-recognisable personal data in
+                    the document before anything reads it: each match
+                    becomes «redacted <rule> <hex>» in the prompt, the
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </hex></rule> ], start tag match failed [ <hex> ] on line 180.

(tag-pair)